src/cmd-sign

```diff
@@ -180,6 +180,7 @@ def robosign_ostree(args, s3, build, gpgkey):
         ostree_image = build['images']['ostree']
         exported_ostree_path = os.path.join(builddir, ostree_image['path'])
         if exported_ostree_path.endswith('.ociarchive'):
+            os.unlink(exported_ostree_path)
```

Comment:
Optional:

```suggestion
            # Delete previous OCI image before re-exporting. Otherwise skopeo will
            # try overwrite the existing file and fail with permission denied.
            os.unlink(exported_ostree_path)
```

I think the context is useful.